src/proto/hierarchy_id.proto

```diff
@@ -13,4 +13,5 @@ enum HierarchyId {
     LapNIG = 4; // Laplace - Normal Inverse Gamma
     FA = 5; // Factor Analysers
     NNxIG = 6; // Normal - Normal x Inverse Gamma
+    Python_h = 7; // Generic python hierarchy
```

Comment:
this is me being picky, but i don't like mixing camelCase with snake_case. I think a better naming would be `PythonHier`